trustwallet/wallet-core
# [Rust]: Add cargo fuzz tests

## Description

Add `cargo fuzz` tests in order to check if `tw_keypair`, `tw_encoding` and `tw_hash` don't crash on arbitrary inputs.

Fixes:
1. `starknet-crypto` - https://github.com/xJonathanLEI/starknet-rs/issues/365
2. `tw_hash::blake2_b*`

## How to test

Fuzzing tests may take too long to find critical bugs, so there is no need to run `cargo fuzz` on CI.
To make sure if all fuzz targets are compilable, run `tools/rust-fuzz dry`.
